encoders/nlp/UniversalSentenceEncoder/tests/test_universalsentenceencoder.py

```diff
@@ -75,3 +75,23 @@ def test_save_and_load_config(mocker):
     encoder_loaded = BaseExecutor.load_config(encoder.config_abspath)
     assert encoder_loaded.model_url == encoder.model_url
     rm_files([encoder.config_abspath])
+
+
+@mock.patch('tensorflow_hub.load', return_value=MockModule())
+def test_get_universal_sentence_encoder(mocker):
+    MODEL_ENCODER = 'https://tfhub.dev/google/universal-sentence-encoder/4'
+    metas = get_metas()
+    encoder = UniversalSentenceEncoder(metas=metas)
+    assert encoder.model_url == MODEL_ENCODER
+
+
+@mock.patch('tensorflow_hub.load', return_value=MockModule())
+def test_get_universal_sentence_encoder_cmlm(mocker):
```

Comment:
@JoanFM  Here we have a test to validate the preprocess parameter. do we need another one?

Reply:
I mean that it actually properly encodes some input, here we are only testing initialization and post_init.